collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -736,6 +736,7 @@ public void run() {
                 String bearing = intent.getStringExtra(BEARING_RESULT);
                 ((ODKView) mCurrentView).setBinaryData(bearing);
                 saveAnswersForCurrentScreen(DO_NOT_EVALUATE_CONSTRAINTS);
+                break;
```

Comment:
I didn't face any problem.
Not adding break after cases is just a bad practice in my opinion as an unnecessary fall-through isn't justified.
It shows up as an warning while you do a code-inspection.